src/core/lib/iomgr/closure.h

```diff
@@ -69,7 +72,9 @@ struct grpc_closure {
    *  space */
   union {
     grpc_closure* next;
-    gpr_mpscq_node atm_next;
+    grpc_core::ManualConstructor<
```

Comment:
Yes, because we still have a lot of places where `grpc_closure` is created using `gpr_malloc()` instead of `New<>()`, in which case its ctor will not be run.  This is exactly the case that `ManualConstructor<>` was designed to address.

As we replace `grpc_closure` with something else, we'll make sure that the new thing is properly instantiated, so it won't need `ManualConstructor<>`.